src/main/java/com/slicingdice/jslicer/SlicingDice.java

```diff
@@ -360,13 +361,13 @@ private JSONObject dataExtractionWrapper(final String url, final JSONObject quer
     }
 
     /**
-     * Get all getProjects in your account.
+     * Get all getDatabase in your account.
```

Comment:
Update this comment correctly.
This should be something like: "Get information about current database."